docs/docsite/rst/mazer/collection.rst

```diff
@@ -0,0 +1,169 @@
+==========================
+Ansible Content Collection
+==========================
+
+The Manifest format
+===================
+
+The `galaxy.yml` for each package is called its manifest. It must be a
+file in `YAML` format that has the structure described below.
+
+.. code-block:: yaml
+
+  name: "geerlinguy.jenkins"
+  version: "3.5.0"
+  authors: ["Jeff Geerling <geerlingguy@mac.com>"]
+  license: "(BSD-3-Clause OR MIT)"
+  description: "Install and configure Jenkins CI server."
+
+  keywords: ["development", "packaging", "jenkins", "ci"]
+
+  readme: "README.md"
+
+  dependencies:
+    "geerlinguy.java": "1.3.0"
+
+Common fields
+-------------
+
+`name`
+^^^^^^
+
+A collection name.
+
+`version`
+^^^^^^^^^
+
+A collection version. Galaxy uses `Semantic Versioning <https://semver.org/>`_
+for collection versions, so you should follow some basic rules:
+
+* Use version numbers with three numeric parts such as `1.0.0` rather than `1.0`.
+
+`authors`
+^^^^^^^^^
+
+The list of collection authors. Each author is a string with a name and
+optionally email and url:
+
+.. code-block:: yaml
+
+  authors:
+    - "Alice Williams <awilliams@example.com>"
+    - "Bob Smith (http://bsmith.example.com)"
+    - "Jack Jackson"
+
+`description`
+^^^^^^^^^^^^^
+
+A short summary description of the collection.
+
+`license`
+^^^^^^^^^
+
+A license of a collection. If you are using a common license such as
+3-Clause BSD or MIT, add a current SPDX license identifier for the license
+you are using:
+
+.. code-block:: yaml
+
+  license: "MIT"
+
+You can use `The SPDX License List <https://spdx.org/licenses/>`_.
+`OSI Approvded <https://opensource.org/licenses/alphabetical>`_ licenses
+are more preferrable.
+
+keywords
+^^^^^^^^
+
+A list of collection keywords.
```

Comment:
Should follow the same criteria as Galaxy tags, where a tag can only contain alphanumeric characters.

Reply:
Done.